modules/toast/react/stories/stories.tsx

```diff
@@ -72,6 +72,11 @@ class ToastWrapper extends React.Component<{state?: string}> {
 storiesOf('Components|Popups/Toast/React', module)
   .addParameters({component: Toast})
   .addDecorator(withReadme(README))
+  .add('Default', () => (
+    <div className="story">
+      <Toast>Your workbook was successfully processed.</Toast>
+    </div>
+  ))
   .add('Successful', () => (
     <div className="story">
       <ToastWrapper state={'Success'} />
```

Comment:
Now that you've added the visual testing story, these are kind of redundant. Can we rename the `Default` story to successful, remove this one, change `Error` to a single case and strip out all of the `ToastWrapper` stuff?